Server/JokesOnYou.Web.Api/Repositories/TagRepository.cs

```diff
@@ -24,8 +26,32 @@ public TagRepository(DataContext context, IMapper mapper)
             _mapper = mapper;
         }
 
-        public async Task<IEnumerable<TagReplyDto>> GetAllTagDtosAsync() =>
-            await _context.Tags.ProjectTo<TagReplyDto>(_mapper.ConfigurationProvider).ToListAsync();
+        public async Task<PaginatedList<TagReplyDto>> GetAllTagDtosAsync(TagFilterDto tagFilterDto)
+        {
+            var query = _context.Tags.AsNoTracking();
+
+            if (!string.IsNullOrWhiteSpace(tagFilterDto.OwnerId))
+            {
+                query = query.Where(t => t.OwnerId == tagFilterDto.OwnerId);
+            }
+            if (tagFilterDto.MinimumLikes != int.MinValue || tagFilterDto.MaximumLikes != int.MaxValue)
```

Comment:
The if statement here can be removed, f.e if the parameter is already int.minValue then all tags will be returned anyways